MiniApp/Classes/core/Protocols/MiniAppMessageDelegate.swift

```diff
@@ -65,6 +66,19 @@ public extension MiniAppMessageDelegate {
         semaphore.wait()
         return uniqueId
     }
+
+    func getHostEnvironmentInfo(completionHandler: @escaping (Result<MAHostEnvironmentInfo, MASDKError>) -> Void) {
+        let environment = Environment(bundle: Bundle.main)
+        guard
+            let sdkVersion = environment.sdkVersion?.description
+        else {
+            completionHandler(.failure(.unknownError(domain: MASDKLocale.localize(.hostAppError), code: 1, description: MASDKLocale.localize(.invalidSDKId))))
+            return
+        }
+        let platformVersion = "iOS " + UIDevice.current.systemVersion
```

Comment:
I don't think we should prepend `iOS` here since we already have the platform name in another field.

Reply:
@corycaywood  ok sure I've removed it